Source/UI/CommonUI/Protocols/Elements/ElementTextFieldStyle.swift

```diff
@@ -10,4 +10,5 @@ public protocol ElementTextFieldStyle: ElementStyle {
     var normalBorderColor: UIColor { get }
     var focusBorderColor: UIColor { get }
     var errorBorderColor: UIColor { get }
+    var sideBorders: [SideBorder]? { get }
```

Comment:
I think at this stage we can group all the border properties into a new object.

```swift
public struct BorderStyle {
    public var cornerRadius: CGFloat
    public var borderWidth: CGFloat
    public var normalBorderColor: UIColor        
    public var focusBorderColor: UIColor
    public var errorBorderColor: UIColor
}
```

This should help improve discoverability and fluid navigation of the properties within.

So after ☝️ we would not have
```swift
public protocol ElementTextFieldStyle: ElementStyle {
    var isSupportingNumericKeyboard: Bool { get set }
    var height: Double { get }
    var borderStyle: BorderStyle { get }
    var placeholder: String? { get }
    var tintColor: UIColor { get }
}
```